src/ONE.Models.CSharp/Common/Configuration/Graphs/GraphConfigurationData.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections.Generic;
+
+namespace ONE.Models.CSharp.Models.Common.Configuration.Graphs
+{
+    public class GraphConfigurationData
+    {
+        public string name { get; set; }
+        public int defaultDateRange { get; set; }
+        public List<int> columns { get; set; }
+        public List<Series> series { get; set; }
+
+        public class Series
```

Comment:
I think there is a Series protobuf object is this related to that in anyway?  If not perhaps it should be renamed?

Reply:
Renamed to avoid confusion.